### PR TITLE
add releasedBuilds() to ProductVersion class

### PR DESCRIPTION
**Pull request by ktdreyer**
Add a method to query all the released builds for this Product Version.

### CODE REVIEW

**Comment by phracek** [comment]
Sorry, I am a bit busy with some container tasks. I will try to test it on Friday or next week.
Thanks for understanding.

File: errata_tool/product_version.py [hunk 1]
```diff
@@ -12,6 +12,22 @@ def __init__(self, id_or_name):
...
+
+        :returns: a (possibly-empty) list of dicts. Each dict represents a
+                  build, for example::
+
+                    {'build': u'ceph-12.2.5-42.el7cp',
+                     'created_at': '2018-09-26T18:17:33Z',
+                     'errata_id': 33840,
+                     'updated_at': '2018-09-26T18:17:33Z'},
+        """
+        url = '/api/v1/product_versions/%d/released_builds' % self.id
```
Comment (phracek) [inline errata_tool/product_version.py]: The URL address is fine, but how can I get `self.id`? Shouldn't be a part of parameter in function releasedBuilds?
I can imagine code like:

~~~
p = Product('RHSCL')
pv = ProductVersion('p.id')
rb = pv.releasedBuilds()
~~~

Am I correct?

Comment (ktdreyer) [reply errata_tool/product_version.py]: That's a great question. We definitely need more documentation / examples for this.

"RHSCL" is a Product, and each product has many Product Versions.

You can initialize the `ProductVersion` class by passing in the PV name. "RHEL-7-RHSCL-3.1" is one that I see active in the ET.

```python
from errata_tool.product_version import ProductVersion
from pprint import pprint
pv = ProductVersion('RHEL-7-RHSCL-3.1')
assert pv.enabled
builds = pv.releasedBuilds()
pprint(builds)
```

**Comment by phracek** [comment]
I guess, we can merge it. LGTM

File: errata_tool/product_version.py (hunk 1 above)
Comment (phracek) [reply errata_tool/product_version.py]: Only for sure.
~~~
p = Product('RHSCL')
pprint(p.data)
~~~
returns
~~~
"build_path": null,
"cdw_flag_prefix": null,
"default_docs_reviewer_id": null,
 "default_solution_id": 2,
"description": "Red Hat Software Collections",
"ftp_path": "",
"ftp_subdir": "RHSCL",
 "id": 83,
 "is_internal": false,
 "isactive": 1,
 "move_bugs_on_qe": false,
 "name": "Red Hat Software Collections",
"path": null,
 "pdc_product_id": null,
 "short_name": "RHSCL",
 "state_machine_rule_set_id": 3,
 "supports_pdc": false,
 "text_only_advisories_require_dists": true,
 "valid_bug_states": [
 "MODIFIED",
  "VERIFIED"
 ]

I don't see any relationships->product_versions here.
Based on https://errata.devel.redhat.com/developer-guide/api-http-api.html#api-releases

Comment (ktdreyer) [reply errata_tool/product_version.py]: I'm working on a patch to add a `productVersions()` method to the `Product` class. I'll need to refresh another fixture though :)

**Pull request merged**